package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "cf-graphql",
   "description": "Generate a GraphQL schema out of your Contentful space",
-  "version": "0.5.0",
```

Comment:
It's a fix, version should be 0.5.1

Reply:
Thanks very much and sorry - I wasn't too sure how this would be handled. I've changed this now.